### DISCUSSION TITLE
Help with upload file from dropdown

### DISCUSSION THREAD

**Discussion by hakeem86** [fomantic/Fomantic-UI#415] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Help Wanted

## Problem
I'm trying access a hidden file upload form from a dropdown box. I had another solution working in that past with javascript: document.getElementById('clickID').click();
But that no longer works for some strange reason. 

My new solution work but I have to add a margin-right: .78571429rem; to the icon, and I have to click within the red border. how can I make the whole area clickable. 

Any help is appreciated. 

My fiddle
https://jsfiddle.net/uwpdafem/1/

**Comment by lubber-de** [comment, marked as answer]
You have to remove the padding from the item and add it to the fileupload label instead.
Adjusted your fiddle here: https://jsfiddle.net/n8gz1oxu/
![image](https://user-images.githubusercontent.com/18379884/51473584-10b62200-1d7d-11e9-927f-742904d57788.png)

**Comment by hakeem86** [comment]
You are awesome ! :+1:
